presto-docs/src/main/sphinx/sql/insert.rst

```diff
@@ -7,18 +7,19 @@ Synopsis
 
 .. code-block:: none
 
-    INSERT INTO table_name query
+    INSERT INTO table_name [ ( column [, ... ] )  query
 
 Description
 -----------
 
 Insert new rows into a table.
 
-.. note::
+When the list of column names is specified the columns produced by the 
+query must exactly match to them. Each column not being present in 
```

Comment:
Reword it slightly:

"Each column in the table not present in the column list ..."